app.js

```diff
@@ -0,0 +1,14 @@
+var express = require('express');
+var truckRoutes= require('./routes/truckRoutes');
+var bodyParser = require('body-parser');
+var app = express();
+
+
+app.use('/trucks', truckRoutes);
```

Comment:
This should appear after the other middle-ware functions. For example, since the bodyParser.urlencoded() middle-ware is used after the router middle-ware, it doesn't actually parse the request body before the .post route handles the request, meaning new trucks are never added to the database.